### ISSUE TITLE
Setting capabilitiesOptions during pairing not working for dim (slider)

### ISSUE THREAD

**Issue by SailingDuck** [athombv/apps-sdk-issues#42]
Better logic numslide device is currently using 0 and 1 as min and max values for the slider. The capabilitiesOptions do not seem to be set. Tried numerous variations none of them seem to work. Even moved to a custom capability as it seemed dim is hardcoded to 0-1, also no luck. 

Can someone take a look at what is happening here?

Example:
https://github.com/PatrickSannes/BetterLogic/blob/SDK2/drivers/numslide/driver.js#L54
(small note: it seemed to be using strings for min/max, but also setting hardcoded numbers to see if the values change from the default in app.json is not working)

**Comment by WeeJeWel** [comment]
This will be fixed in the next Homey app release.

It has already been fixed on developer.athom.com.

**Comment by SailingDuck** [comment]
Tested this by pairing on developer.athom.com. Good thin is, the values can be set. 
Weird thing is I can't get the slider fully down anymore. I set it from 100-200. Below screenshot is a console log when I drag it to the max and down to the min. As seen in the screenshot is the lowest I can drag it too. Does it require an app update to apply the options to the slider?

![image](https://user-images.githubusercontent.com/23644634/51565936-962bf600-1e93-11e9-990c-82b847bda414.png)

Output of the API playground:
![image](https://user-images.githubusercontent.com/23644634/51566313-8e208600-1e94-11e9-8507-762e14107d4e.png)

A little lower the property capabilitiesOptions is undefined.
`"capabilitiesOptions":undefined`

Is this correct, or do I need to wait for an app update?
